data/json/items/ammo.json

```diff
@@ -179,6 +179,7 @@
         "weight" : 80,
         "ammo_type" : "plutonium",
         "count" : 5,
+        "energy": 200,
```

Comment:
It's per effective unit of plutonium where 1 putonium cell = 500 units = 100'000 energy. That plutonium charges multiplier always causes problems but I don't have any better solution at this time